### ISSUE TITLE
Rails/SkipsModelValidations is inconsistence with touch

### ISSUE THREAD

**Issue by Ana06** [rubocop-hq/rubocop#4098]
Rails/SkipsModelValidations cop doesn't complain about the following relation:

`belongs_to :user, touch: true`

but it does complain in its test (_Avoid using touch because it skips validations._):

`it { should belong_to(:user).touch(true) }`

## Expected behavior

Complain in both cases or in any of them. I would say that it should also complain in the relation.

## RuboCop version

```
0.47.1
```

**Comment by bbatsov** [comment]
Guess you're right.

**Comment by bbatsov** [comment]
We're in the process of moving all Rails-related functionality to a standalone library (gem) named `rubocop-rails`. Please, migrate this issue to its issue tracker https://github.com/rubocop-hq/rubocop-rails/issues/
